webknossos/lint.sh

```diff
@@ -1,8 +1,4 @@
 #!/usr/bin/env bash
 set -eEuo pipefail
 
-# Pylint doesn't lint files in directories that don't have an __init__.py
-# This will maybe be fixed by https://github.com/PyCQA/pylint/issues/352
-# In the meantime, find all python files, except for the ./webknossos/client/_generated directory and lint them
-# Inspired by https://stackoverflow.com/questions/4210042/how-to-exclude-a-directory-in-find-command
-find . -type d \( -path ./webknossos/client/_generated -o -path ./tests/.webknossos-server -o -path ./stubs \) -prune -o -iname "*.py" -print | xargs poetry run python -m pylint -j2
+poetry run ruff check --fix .
```

Comment:
This command is already included in the `./format.sh` command. Perhaps we should get rid of this file entirely. What do you think?

Reply:
@philippotto Any thoughts on this?

Reply:
I just commented [here](https://github.com/scalableminds/webknossos-libs/pull/1013/files/d25afc9bc1ca7d45fbd6888ca552ce1ac8d955c2#r1524619872). if it's not possible to separate linting and formatting, a single file would make sense (`format-and-lint.sh`).